patroni/utils.py

```diff
@@ -1180,24 +1180,24 @@ def unquote(string: str) -> str:
     return ret
 
 
-def get_major_version(bin_dir: Optional[str] = None, bin_name: str = 'postgres') -> str:
-    """Get the major version of PostgreSQL.
+def get_postgres_version(bin_dir: Optional[str] = None, bin_name: str = 'postgres') -> str:
+    """Get full PostgreSQL version.
 
-    It is based on the output of ``postgres --version``.
+    Like func:`get_major_version` but extended with minor version.
```

Comment:
I think this line should be swapped with https://github.com/patroni/patroni/pull/3130/files#diff-1b3584607a3bcb7f81de8462e86c83f6ec566b0f0d2b58eb969f1ec4a778e7c0R1224 (and slightly modified)